composer.json

```diff
@@ -6,7 +6,7 @@
     "minimum-stability": "dev",
     "prefer-stable": true,
     "require": {
-        "drupal/cas": "dev-1.x#7c40fc180d462311c2cb5af5c0f43a9043356e64",
+        "drupal/cas": "^1.5",
```

Comment:
Hi @dxvargas, the latest updates from CAS brought changes to the redirection logic after logging in. This means that our current solution doesn't work (and thats why the tests are failing). I think this update should go in a separate PR with the fixes required to the redirectio nlogic